src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_SIZE = 101;
+    private static final int MAX_RADIUS = 51;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int SET_OF_FIGURES = 5;
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+    private final Random rand = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    private int getAttribute() {
+        return rand.nextInt(MAX_SIZE);
+    }
+
+    public Figure getRandomFigure() {
+        Figure figure;
+        int index = rand.nextInt(SET_OF_FIGURES);
+        int radius = rand.nextInt(MAX_RADIUS);
+        int size = getAttribute();
+        int base = getAttribute();
+        int height = getAttribute();
+        int leg = getAttribute();
+        String color = colorSupplier.getRandomColor();
+
+        switch (index) {
+            case 0:
+                figure = new Circle(radius, color);
+                break;
+            case 1:
+                figure = new Square(color, size);
```

Comment:
```suggestion
                return new Square(color, getAttribute());
```

same for other figures